### PR TITLE
ui: combine jobs and schedules page

### PR DESCRIPTION
**Pull request by maryliag**
Note to Reviewers: Only second commit is relevant to this PR

---
The new Schedules page is now grouped on the Jobs page.
This page only exists on DB Console and it not available
on CC Console.

Part of #70725

https://www.loom.com/share/134c2296a97a4eb5941cf54a5d162a63

Screen Shot 2022-08-26 at 10 55 47 AM

Screen Shot 2022-08-26 at 10 55 54 AM

Release justification: low risk change
Release note (ui change): Combine the Jobs page and the new added
Schedules page into one. This is only available on DB Console.

### CODE REVIEW

**Review by benbardin — COMMENTED** [review]
I love how this looks, thank you!!

File: pkg/ui/workspaces/db-console/src/app.tsx [hunk 1]
```diff
@@ -160,6 +161,11 @@ export const App: React.FC<AppProps> = (props: AppProps) => {
                   <Route path="/events" component={EventPage} />
                   <Route exact path="/jobs" component={JobsPage} />
                   <Route path={"/jobs/:id"} component={JobDetails} />
+                  <Route
+                    exact
+                    path="/jobs-schedules"
+                    component={JobsAndSchedulesPage}
+                  />
 
                   <Route exact path="/schedules" component={SchedulesPage} />
```
[the remark this answers is not in the document]
Comment (maryliag) [reply pkg/ui/workspaces/db-console/src/app.tsx]: done

**Comment by livlobo** [comment]
@maryliag  @benbardin - I am slightly concerned that the 'schedules' page is not discoverable from the Left-hand navigation and a bit hidden away in the tab. Is there a recommended pattern to make it more discoverable for users who don't know that schedules are implemented as "jobs" under the hood? 

Jobs are really an implementation detail and IMO users won't have a mental model of --> I want to see what's going on with my backup schedule, that schedule is implemented as a job, I should find it nested under the jobs section in the console. 

cc: @thtruo @jeremycbloom

**Comment by adityamaru** [comment]
> Jobs are really an implementation detail and IMO users won’t have a mental model of --> I want to see what’s going on with my backup schedule, that schedule is implemented as a job, I should find it nested under the jobs section in the console.

drive-by comment but I would like to second wha. It really depends on what we’re using the term “Jobs” in left hand column to mean here. If they refer to how we use the term in code then a schedule is not a job, a schedule spawns jobs so the relationship seems inverted in the consolidated view.

**Comment by kevin-v-ngo** [comment]
> @maryliag @benbardin - I am slightly concerned that the 'schedules' page is not discoverable from the Left-hand navigation and a bit hidden away in the tab. Is there a recommended pattern to make it more discoverable for users who don't know that schedules are implemented as "jobs" under the hood?
> 
> Jobs are really an implementation detail and IMO users won't have a mental model of --> I want to see what's going on with my backup schedule, that schedule is implemented as a job, I should find it nested under the jobs section in the console.
> 
> cc: @thtruo @jeremycbloom

What if in the short-term, we called it "Jobs & Schedules"? In the longer term, I really think we should reconsider the Jobs page entirely. As Liv mentioned, it’s an implementation detail and requires userers to know what is and isn’t a job. 

We should be more user-journey oriented with the the left nav. I would even consider Schedules as an implementation detail as well. What if we introduced more Schedules the future? I think TTL is a schedule?

For instance, I’d prefer specific nav items such as “Backup & Restore", "Business Continuity", or "Disaster recovery” and  “Change Feeds” for CDC jobs. It would also allow us to create more custom and tailored observability experiences that's required for these user journeys.

**Comment by maryliag** [comment]
can I close this? since it doesn't look like we're making this change
cc @thtruo

**Comment by benbardin** [comment]
Yes, please close. Thank you anyway! I'll keep this bookmarked to see how to do tabs in the future :)

**Comment by kevin-v-ngo** [comment]
FYI @jeremycbloom Here's the current nav for 22.2:

image

**Comment by jeremycbloom** [comment]
Thanks, Kevin. I'm thinking of a starter project for our new senior designer that could focus on a navigation structure audit.

**Pull request closed without merging**